parquet-avro/src/test/java/parquet/avro/TestAvroReadSupport.java

```diff
@@ -0,0 +1,198 @@
+package parquet.avro;
+
+import java.util.*;
+import org.apache.avro.Schema;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import parquet.hadoop.api.InitContext;
+import parquet.schema.MessageType;
+import parquet.schema.MessageTypeParser;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static parquet.avro.AvroReadSupport.*;
+
+public class TestAvroReadSupport {
+
+  /* Avro Schemas */
+
+  /* equivalent to message Foo { required binary field_1 (UTF8); } */
+  static final String RECORD_WITH_REQUIRED_STRING = "{\"type\":\"record\",\"name\":\"Foo\",\"fields\":[{\"name\":\"field_1\",\"type\":{\"type\":\"string\"}}]}";
```

Comment:
Please break these across multiple lines so they are more readable. (Max line length is 80.)